src/pages/costModels/components/inputs/selector.tsx

```diff
@@ -94,6 +94,7 @@ const SelectorBase: React.FunctionComponent<SelectorProps> = ({
     >
       <Select
         id={id}
+        maxHeight={350}
```

Comment:
There is `maxHeight` and other variables stored in wizard.styles. Can we add this value there?

Reply:
I was applying the maxHeight to any use of the select dialog. Right now wizard styles are being used by the generalInformation component only. They are also always being used as real styles (passed into `style` attributes on basic HTML elements), this value is a custom PF prop that gets handled by PF rather than as an actual style attribute.

If desired, I could create a maxHeight prop on our Selector component that we then pass into the PF Select component.. and just pass it to our Selector component from generalInformation with a `maxHeight={styles.selector.maxHeight`.

Reply:
Yes, `maxHeight={styles.selector.maxHeight}` is what I was looking for. We typically place all style properties, like height in a `styles` file -- even though PatternFly may treat this one differently. Easier to find properties and / or reuse them when located together.

I agree that a `maxHeight` prop would be best, so we can use it only when necessary.